### ISSUE TITLE
Modules intro page code example does not build

### ISSUE THREAD

**Issue by chrisalexander** [antirez/redis-doc#1248]
While I appreciate there are a lot of other examples elsewhere, the example on the page https://redis.io/topics/modules-intro does not build:

```
#include "redismodule.h"
#include <stdlib.h>

int HelloworldRand_RedisCommand(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
    RedisModule_ReplyWithLongLong(ctx,rand());
    return REDISMODULE_OK;
}

int RedisModule_OnLoad(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
    if (RedisModule_Init(ctx,"helloworld",1,REDISMODULE_APIVER_1)
        == REDISMODULE_ERR) return REDISMODULE_ERR;

    if (RedisModule_CreateCommand(ctx,"helloworld.rand",
        HelloworldRand_RedisCommand) == REDISMODULE_ERR)
        return REDISMODULE_ERR;

    return REDISMODULE_OK;
}
```

RedisModule_CreateCommand actually has 7 arguments (rather than 3) in all versions I could find anywhere.

I think changing that line to this would be sufficient, although I'm not sure exactly what the argument "readonly" is doing, although I guess that option is appropriate. Not sure what the numbers mean either at this point, but I got the examples from https://github.com/antirez/neural-redis/blob/master/neuralredis.c.

```
if (RedisModule_CreateCommand(ctx,"helloworld.rand",
            HelloworldRand_RedisCommand,"readonly",1,1,1) == REDISMODULE_ERR)
        return REDISMODULE_ERR;
```

**Comment by itamarhaber** [comment]
Hello @chrisalexander 

Thanks for reporting the issue. The example is indeed wrong (it uses a prehistoric version of the modules API) - should be resolved via #1249.

**Comment by chrisalexander** [comment]
Ace, thanks very much!
